lib/rubocop-rspec_rails.rb

```diff
@@ -4,9 +4,6 @@
 require 'yaml'
 
 require 'rubocop'
-
-require 'rubocop/rspec/language/node_pattern'
```

Comment:
We can merge this PR because we don’t need the Language::NodePattern, and we can rely on both rubocop-rspec that has it or has not, no difference. Or am I missing something?

Reply:
If I revert the change to Gemfile (and keep `gem 'rubocop-rspec', '~> 2.27'`), CI starts failing with this stacktrace:
```
TypeError: wrong argument type Class (expected Module) (TypeError)

      extend NodePattern
             ^^^^^^^^^^^
/home/runner/work/rubocop-rspec_rails/rubocop-rspec_rails/vendor/bundle/ruby/3.2.0/gems/rubocop-rspec-2.29.2/lib/rubocop/rspec/language.rb:16:in `extend'
/home/runner/work/rubocop-rspec_rails/rubocop-rspec_rails/vendor/bundle/ruby/3.2.0/gems/rubocop-rspec-2.29.2/lib/rubocop/rspec/language.rb:16:in `<module:Language>'
/home/runner/work/rubocop-rspec_rails/rubocop-rspec_rails/vendor/bundle/ruby/3.2.0/gems/rubocop-rspec-2.29.2/lib/rubocop/rspec/language.rb:14:in `<module:RSpec>'
/home/runner/work/rubocop-rspec_rails/rubocop-rspec_rails/vendor/bundle/ruby/3.2.0/gems/rubocop-rspec-2.29.2/lib/rubocop/rspec/language.rb:4:in `<module:RuboCop>'
/home/runner/work/rubocop-rspec_rails/rubocop-rspec_rails/vendor/bundle/ruby/3.2.0/gems/rubocop-rspec-2.29.2/lib/rubocop/rspec/language.rb:3:in `<top (required)>'
<internal:/opt/hostedtoolcache/Ruby/3.2.4/x64/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:38:in `require'
<internal:/opt/hostedtoolcache/Ruby/3.2.4/x64/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:38:in `require'
/home/runner/work/rubocop-rspec_rails/rubocop-rspec_rails/lib/rubocop-rspec_rails.rb:7:in `<top (required)>'
<internal:/opt/hostedtoolcache/Ruby/3.2.4/x64/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:38:in `require'
<internal:/opt/hostedtoolcache/Ruby/3.2.4/x64/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:38:in `require'
tasks/cops_documentation.rake:4:in `<top (required)>'
/home/runner/work/rubocop-rspec_rails/rubocop-rspec_rails/Rakefile:19:in `load'
/home/runner/work/rubocop-rspec_rails/rubocop-rspec_rails/Rakefile:19:in `block in <top (required)>'
/home/runner/work/rubocop-rspec_rails/rubocop-rspec_rails/Rakefile:19:in `each'
/home/runner/work/rubocop-rspec_rails/rubocop-rspec_rails/Rakefile:19:in `<top (required)>'
/home/runner/work/rubocop-rspec_rails/rubocop-rspec_rails/vendor/bundle/ruby/3.2.0/gems/rake-13.2.1/exe/rake:27:in `<top (required)>'
/opt/hostedtoolcache/Ruby/3.2.4/x64/bin/bundle:25:in `load'
/opt/hostedtoolcache/Ruby/3.2.4/x64/bin/bundle:25:in `<main>'
```

Example run: https://github.com/rubocop/rubocop-rspec_rails/actions/runs/9161367523/job/25186054266